docs/deploy/cloud.md

```diff
@@ -0,0 +1,170 @@
+---
+sidebar_position: 4
+description: "Learn how to use Restate Cloud."
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import Admonition from '@theme/Admonition';
+
+# Restate Cloud
+
+<Admonition type="warning">
+    Restate Cloud is an early-access preview. Features may change, and we offer
+    no reliability SLAs at this time. Production usage is not recommended.
+</Admonition>
+
+This page describes how to use Restate environments deployed on our early-access
```

Comment:
Can we first add a sentence on what Restate Cloud actually offers? Something like `Restate Cloud is the managed service deployment of Restate. You deploy your services on Lambda or... When you create an account, you can create an environment.... An environment is.... `